testnet/stacks-node/src/config.rs

```diff
@@ -569,11 +571,105 @@ impl Config {
                     rbf_fee_increment: burnchain
                         .rbf_fee_increment
                         .unwrap_or(default_burnchain_config.rbf_fee_increment),
-                    epochs: match burnchain.epochs {
-                        Some(epochs) => Some(epochs),
-                        None => default_burnchain_config.epochs,
-                    },
+                    epochs: default_burnchain_config.epochs,
+                };
+
+                if let Some(ref conf_epochs) = burnchain.epochs {
+                    let default_epochs = match result.get_bitcoin_network().1 {
```

Comment:
Looking at the the get_bitcoin_network function, the string arg here might be able to be dropped ultimately since it seems to be duplicating what's in the enum. 

Barring that though, maybe let's change this to the following for clarity:

```suggestion
                    let (_, default_epochs) = match result.get_bitcoin_network()
                    match default_epochs {
```

Reply:
Yeah, I think I agree regarding eliminating the first return element of `get_bitcoin_network()`, but I try to avoid refactoring functions that aren't directly related to the PR. Regarding your second suggestion: I refactored this code so that the enum is now a function argument to a constructor for the new configuration settings, so I think that improves the clarity of what's happening.